crates/sncast/tests/helpers/fixtures.rs

```diff
@@ -537,3 +540,37 @@ pub fn get_keystores_path(relative_path_from_cargo_toml: &str) -> String {
         .expect("Failed to convert path to string")
         .to_string()
 }
+
+pub fn assert_tx_entry_failed(
+    tx_entry: &ScriptTransactionEntry,
+    name: &str,
+    status: ScriptTransactionStatus,
+    msg_contains: Vec<&str>,
+) {
+    assert_eq!(tx_entry.name, name);
+    assert_eq!(tx_entry.status, status);
+
+    let ScriptTransactionOutput::ErrorResponse(response) = tx_entry.output.clone() else {
+        panic!("Wrong response")
+    };
+    for msg in msg_contains {
+        assert!(response.message.contains(msg));
+    }
+
+    assert!(tx_entry.timestamp > 1_709_853_748);
```

Comment:
Can we define a constant for this number?

Reply:
sure, named it `SCRIPT_ORIGIN_TIMESTAMP`, but if you have a better idea - please share